jenkins-design-language/src/js/components/Icon.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { PropTypes } from 'react';
-import * as IconId from './material-ui/svg-icons';
+import * as IconId from '@jenkins-cd/blueocean-material-icons';
```

Comment:
The way I made the blueocean-material-icons package is that it exports all the icons individually because they do not change, this also means anyone can use the icons separately if needed (and not have to use the Icon component). Also the Icon component is how we use these icons in blueocean, making it part of the JDL in my opinion, this component is what we will continue to tweak in the future and I wanted to keep in the JDL because of that. Also this way it is very easy to iterate through all the icons for the Icon stories so we have a way to see them all in one place.

Reply:
Cool, that works for me 👍